io.github.wereturtle.ghostwriter.json

```diff
@@ -0,0 +1,120 @@
+{
+    "app-id": "io.github.wereturtle.ghostwriter",
+    "base": "io.qt.qtwebkit.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "ghostwriter",
+    "rename-icon": "ghostwriter",
+    "rename-desktop-file": "ghostwriter.desktop",
+    "rename-appdata-file": "ghostwriter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "cmark",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/commonmark/cmark.git",
+                    "tag": "0.28.3"
+                }
+            ]
+        },
+        {
+            "name": "cmark-gfm",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/github/cmark.git",
+                    "tag": "0.28.3.gfm.12"
+                }
+            ]
+        },
+        {
+            "name": "MultiMarkdown",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/fletcher/MultiMarkdown-6.git",
+                    "tag": "6.3.2"
+                }
+            ]
+        },
+        {
+            "name": "discount",
+            "no-autogen": true,
+            "sources":[
+                {
+                    "type":"archive",
+                    "url": "https://github.com/Orc/discount/archive/v2.2.3a.tar.gz",
+                    "sha256": "40defe460dc005dbc78a4e6174fce29fca2ac3ccd5acd400c05eb713f824df97"
+                },
+                {
+                    "type": "shell",
+                    "commands": ["./configure.sh --prefix=/app"]
+                }
+            ]
+        },
+        {
+            "name": "Pandoc",
+            "only-arches": ["x86_64"],
```

Comment:
Is pandoc only for some additional features?

Reply:
Yes, it's for the HTML previews and export.